src/app/game.ts

```diff
@@ -23,6 +23,10 @@ import {
   Resource,
   Fractions,
 } from './models'
+import { Barracks } from './models/buildings/barracks'
+import { Quarry } from './models/buildings/quarry'
```

Comment:
noo, just export from models

Reply:
yeah, I made it too fast and I've forgotten